lib/Compilation.js

```diff
@@ -362,13 +362,13 @@ class Compilation extends Tapable {
 	_addModuleChain(context, dependency, onModule, callback) {
 		const start = this.profile && Date.now();
 
-		const errorAndCallback = this.bail ? function errorAndCallback(err) {
+		const errorAndCallback = this.bail ? (err) => {
 			callback(err);
-		} : function errorAndCallback(err) {
```

Comment:
Oh bahh I totally missed the initial assignment. Yes this makes sense and I'm down with that.